docs/developers/deploy-on-polygon.md

```diff
@@ -90,12 +90,12 @@ wss://polygon-mumbai.infura.io/v3/{projectId}
 If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
 
 ```bash
-https://polygon-mumbai.infura.io/v3/{projectId}
+https://polygon-mainnet.infura.io/v3/{projectId}
 ```
 
 Run the deployment script with your specific parameters.
 ```bash
-node index.js --gasprice 80 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0xD732393f8eC57644675c104Ec8A4661db58CCE41 --pairName " Your token pair name" --expirationTimestamp 1630447200 --collateralPerPair 250000000000000000 --priceIdentifier UMAUSD --longSynthName "long token name" --longSynthSymbol rtUMA-0821 --shortSynthName "Ushort token name" --shortSynthSymbol rtUMA-0821s --collateralToken 0x0000000000000000000000000000000000001010 --customAncillaryData "twapLength:3600" --optimisticOracleLivenessTime 3600 --fpl RangeBond
+node index.js --gasprice 80 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x57EE47829369e2EF62fBb423648bec70d0366204 --pairName " Your token pair name" --expirationTimestamp 1630447200 --collateralPerPair 250000000000000000 --priceIdentifier UMAUSD --longSynthName "long token name" --longSynthSymbol rtUMA-0821 --shortSynthName "Ushort token name" --shortSynthSymbol rtUMA-0821s --collateralToken 0x0000000000000000000000000000000000001010 --customAncillaryData "twapLength:3600" --optimisticOracleLivenessTime 3600 --fpl RangeBond
```

Comment:
Looks like this creator is the mumbai creator, but you changed the node to mainnet. 

I think generally it is better to have the tutorial run through a testnet deployment. Maybe we should update the tutorial to use Kovan - since that is probably easier for the average user than mumbai.